prosper/common/prosper_logging.py

```diff
@@ -124,11 +124,21 @@ def close_handles(self):
                 #TODO:testable?
                 pass #do not crash if can't close handle
 
+    def _configure_common(self, prefix, fallback_level, fallback_format):
+        log_level = self.config.get_option('LOGGING', prefix + 'log_level', None, fallback_level)
+        log_format_name = self.config.get_option('LOGGING', prefix + 'log_format', None, None)
+        if log_format_name:
```

Comment:
Could change this out for a ternary: `log_format = ReportingFormats[log_format_name].value if log_format_name else fallback_format`